common/mount.py

```diff
@@ -615,7 +615,7 @@ def mounted(self):
             return False
 
     def createMountStructure(self):
-        """
+        r"""
```

Comment:
This could work but I wouldn't treat docstrings as r-strings.

Just replace backslashes.
This ` |\ <pid>.lock    ` should be  |- <pid>.lock    `.

Sorry for commenting. Currently I see not technical way to commit something into a "foreign" PR.